core/src/main/java/com/github/kaktushose/jda/commands/internal/register/TreeNode.java

```
@@ -62,7 +63,7 @@ public void addChild(String[] labels, SlashCommandDefinition command) {
         }
         // framework error, SlashCommandDefinition should have prevented this
         if (labels.length > 3) {
-            throw new IllegalArgumentException(
+            throw new JDACException.Internal(
                     "Failed to add child command: \"%s\". Cannot add a child with more than 3 labels! ".formatted(command.displayName()) +
                             "Please report this error to the devs of jda-commands."
```

"Please report this error to the devs of jda-commands." can be removed, would be duplicated